exercises/palindrome-products/Makefile

```diff
@@ -1,9 +1,9 @@
 default: clean test
 
-test:
-	jbuilder runtest
+test: 
+	dune runtest
 
 clean:
-	jbuilder clean
+	dune clean
```

Comment:
I'm not actually experienced with OCaml building; does this mean we're still using `jbuilder`, but we're calling `dune clean` which then calls `jbuilder clean` indirectly, or does it mean we're not using `jbuilder`?

Reply:
Unfortunately I am not experienced with OCaml building either. As far as I understood `dune` is the successor of `jbuilder` with the last version of `jbuilder` emitting messages that direct users to switch to dune.

Reply:
Ah, so we can create an issue for cleaning up and removing references to `jbuilder` separately.